packages/mgt-components/src/components/mgt-todo/mgt-todo.ts

```diff
@@ -280,95 +335,50 @@ export class MgtTodo extends MgtTasksBase {
    * @memberof MgtTodo
    */
   protected renderTask(task: TodoTask) {
-    const context = { task, list: this._currentList };
+    const context = { task, list: this.currentList };
 
     if (this.hasTemplate('task')) {
       return this.renderTemplate('task', context, task.id);
     }
 
     const isCompleted = (TaskStatus as any)[task.status] === TaskStatus.completed;
-    const isLoading = this._loadingTasks.includes(task.id);
-    const taskCheckClasses = {
-      Complete: !isLoading && isCompleted,
-      Loading: isLoading,
-      TaskCheck: true,
-      TaskIcon: true
-    };
 
-    const taskCheckContent = isLoading
-      ? html`
-          
-        `
-      : isCompleted
+    let taskDetailsTemplate = null;
+
+    const taskDueTemplate = task.dueDateTime
       ? html`
-          
+          <fluent-button appearance="stealth" class="TaskDue">
+            <span part="start" class="TaskDueCalendar">${getSvg(SvgIcon.Calendar)}</span>
+            <span part="content" class="TaskDueDate">${getDateString(new Date(task.dueDateTime.dateTime))}</span>
+        </fluent-button>
         `
       : null;
```

Comment:
```suggestion
      : html``;
```

Reply:
null, nothing or '' work well here, the only downside to using them is the return type becomes a union type and not the singular `TemplateResult<T>` but as we're not defining the return type explicitly that works fine.